modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/FeaturedMediaFormatter.php

```diff
@@ -103,7 +107,13 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
       '#options' => image_style_options(FALSE),
       '#description' => t('Image style to be used if the Media is an image.'),
     ];
-
+    $element['view_mode'] = [
```

Comment:
Should we make this setting mandatory?

Reply:
We have now a default value, could that be enought?